prow/config/jobs_test.go

```diff
@@ -95,16 +95,12 @@ func TestPresubmits(t *testing.T) {
 			if !job.SkipReport && job.Context == "" {
 				t.Errorf("Job %s needs a context.", job.Name)
 			}
-			if job.RerunCommand == "" || job.Trigger == "" {
```

Comment:
@cjwagner AFAIK we have defaulting for these anyways now?

Reply:
Yes, but defaulting happens at config load time so these should not be empty at this point.

Reply:
😂 maybe easier just remove the entries? it's easy to regenerate them though

Reply:
that must be broken then? because this tests was failing.

Reply:
actually it's just another case where we fail to actually treat config as config, our types mismatch. fantastic.

I'll revert this and remove the jobs entirely in a moment, have to run to a quick meeting >.<